pages/common/gulp.md

```diff
@@ -0,0 +1,12 @@
+# gulp
+
+> JavaScript task runner and streaming build system.
+> Tasks are defined within gulpfile.js at the project root.
```

Comment:
@waldyrious I moved the gulpfile bit up into the description and shortened the commands description. I made that change according to @rubenvereecken note and you agreeing with him. Is that correct for you?